admin/plugins.php

```diff
@@ -143,13 +157,17 @@ function perflab_render_plugin_card( array $plugin_data ) {
 		case 'install':
 			if ( $status['url'] ) {
 				if ( $compatible_php && $compatible_wp && current_user_can( 'install_plugins' ) ) {
+					$standalone_plugins_file_map = perflab_get_standalone_plugins_file_map();
+					$plugin_file                 = isset( $standalone_plugins_file_map[ $plugin_data['slug'] ] ) ? wp_create_nonce( 'perflab_activate_plugin_' . $standalone_plugins_file_map[ $plugin_data['slug'] ] ) : '';
```

Comment:
See my comment in https://github.com/WordPress/performance/pull/900/files#r1425820667: We could use the plugin slug here and use that same value to validate the nonce later. No need to use the plugin basename.